appyx-interactions/android/src/main/kotlin/com/bumble/appyx/interactions/testing/Utils.kt

```diff
@@ -16,52 +15,9 @@ import androidx.compose.ui.test.junit4.ComposeContentTestRule
 import com.bumble.appyx.interactions.core.model.BaseInteractionModel
 import com.bumble.appyx.interactions.core.ui.helper.InteractionModelSetup
 import com.bumble.appyx.interactions.sample.Children
-import com.bumble.appyx.interactions.sample.SpotlightUi
-import com.bumble.appyx.interactions.sample.TestDriveUi
 import com.bumble.appyx.interactions.theme.appyx_dark
-import com.bumble.appyx.transitionmodel.spotlight.Spotlight
-import com.bumble.appyx.transitionmodel.testdrive.TestDrive
-import com.bumble.appyx.transitionmodel.testdrive.TestDriveModel
 import kotlin.random.Random
 
-fun <InteractionTarget : Any> ComposeContentTestRule.setupSpotlight(
-    spotlight: Spotlight<InteractionTarget>,
-) {
-    setContent {
-        Surface(
-            modifier = Modifier.fillMaxSize(),
-            color = appyx_dark
-        ) {
-            InteractionModelSetup(spotlight)
-
-            SpotlightUi(
-                spotlight = spotlight,
-                color = Color.Blue
-            )
-        }
-    }
-}
-
-@OptIn(ExperimentalMaterialApi::class)
-fun <InteractionTarget : Any> ComposeContentTestRule.setupTestDrive(
-    testDrive: TestDrive<InteractionTarget>,
-    testDriveModel: TestDriveModel<InteractionTarget>,
-) {
-    setContent {
-        Surface(
-            modifier = Modifier.fillMaxSize(),
-            color = appyx_dark
-        ) {
-            InteractionModelSetup(testDrive)
-
-            TestDriveUi(
-                testDrive = testDrive,
-                model = testDriveModel
-            )
-        }
-    }
-}
-
 fun <InteractionTarget : Any, ModelState : Any> ComposeContentTestRule.setupInteractionModel(
     interactionModel: BaseInteractionModel<InteractionTarget, ModelState>,
     fraction: Float = 1.0f,
```

Comment:
`TestChildrenUi` refers to a type parameter as `NavTarget`. Shouldn't we rename it to `InteractionTarget`?